docs/examples/metrics/instruments/requirements.txt

```diff
@@ -0,0 +1,3 @@
+opentelemetry-api~=1.25.0
```

Comment:
Probably want to do `opentelemetry-api~=1.25` not `opentelemetry-api~=1.25.0`. This will help catch any bugs we release in minor versions.

Reply:
done! 👍